### DISCUSSION TITLE
Open a browser fullscreen to frigate

### DISCUSSION THREAD

**Discussion by zeusraman** [gokrazy/gokrazy#283] [Q&A]
hello all,
 First of all just came across and am in love with gokrazy as its light weight and looks amazing
I would like to connect a pizero2w to hdmi for monitoring frigate camera .
whats the most light weight solution using go krazy for it to start and display my url ?

cheers and many many thanks to the devs.

**Comment by damdo** [comment]
Hey @zeusraman try check out my gokrazy kiosk project https://github.com/damdo/gokrazy-kiosk
